.github/workflows/update-version.yml

```diff
@@ -0,0 +1,24 @@
+name: Check TimeZone Version
+
+on:
+  schedule:
+    - cron: '0 0 * * 0'
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup .NET Core
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 5.0.102
+      - name: Build with dotnet
+        run: dotnet build src/System.Runtime.TimeZoneData /t:UpdateToLatestVersion
+      - name: Create Pull Request
+        id: cpr
+        uses: peter-evans/create-pull-request@v3    
```

Comment:
I think we don't allow third-party GitHub Actions by default. We could replace this with a call to the [`gh` command](https://cli.github.com/manual/gh_pr_create) which is preinstalled on the ubuntu-20.04 machines.

Reply:
AFAIK we do for PRs but not for official builds (which don't run on GH anyway). In this case though I would prefer the gh command if it provides the required functionality.